File: Inventor-Training-Material-0c232f96f36e1e0bed31b07a6a6ebb901a31dd40/Module 12 - Apprentice/ApprenticeDemo.VB.Net/ApprenticeDemo/ApprenticeDemo/Form2.vb
Public Class Form2
    Public Sub New(ByVal filename As String)

        ' This call is required by the Windows Form Designer.
        InitializeComponent()

        ' Add any initialization after the InitializeComponent() call.
        AxInventorViewControl1.FileName = filename

    End Sub


    Private Sub FormViewer_SizeChanged(ByVal sender As Object, ByVal e As System.EventArgs) Handles Me.SizeChanged

        AxInventorViewControl1.Size = Me.Size

    End Sub
End Class

File: Inventor-Training-Material-0c232f96f36e1e0bed31b07a6a6ebb901a31dd40/Module 12 - Apprentice/ApprenticeDemo.C#/ApprenticeDemo/ApprenticeDemo/Form2.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace ApprenticeDemo
{
    public partial class Form2 : Form
    {
        public Form2()
        {
            InitializeComponent();
        }

        public  Form2(string filename)
        {
        
        //This call is required by the Windows Form Designer.
        InitializeComponent();

        //Add any initialization after the InitializeComponent() call.
        axInventorViewControl1.FileName = filename;
        }
        private void Form2_Load(object sender, EventArgs e)
        {
            axInventorViewControl1.Size = this.Size;
        }
    }
}
